modules/nw-route-specific-annotations.adoc

```diff
@@ -70,6 +70,21 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 Environment variables cannot be edited.
 ====
 
+.Router timeout variables
+
+TimeUnits are represented by a number followed by the unit: us *(microseconds), ms (milliseconds, default), s (seconds), m (minutes), h *(hours), d (days).
+
+The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
+[cols="3*"]
+|===
+| ROUTER_THREADS | 4s | Specifies the number of threads for the haproxy router.
+| ROUTER_DEFAULT_TUNNEL_TIMEOUT | 1h | Length of time for TCP or WebSocket connections to remain open. If you have websockets/tcp connections (and any time HAProxy is reloaded), the old HAProxy processes will stay for that period.
```

Comment:
If you can, try to pull your information out of the parentheses and include it as a part of the sentence. 

From the IBM style guide:

> Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using parentheses in general text.

Reply:
Acked. Removed the parentheses